docs/packaging.md

```diff
@@ -250,37 +250,59 @@ apt install aziot-identity-service
 aziot init
 ```
 
-The user installs the package, then runs `aziot init` to set up the configuration with minimal information like the device provisioning method.
+After installng the package, run `aziot init` to interactively set up the configuration with minimal information like the device provisioning method.
```

Comment:
run 'iotedge init'

Reply:
This particular example would be if someone were choosing to not use IoT Edge and planned to only install the identity service for other purposes.

Reply:
I thought we plan to enable the globally unified tool in the future. 

If aziot init is still valid, it would be helpful to elaborate it crisp clear here.

Reply:
I changed the section name to be "Installation procedure for non-IoT Edge (`aziot-identity-service` only)".  I also moved it to appear after the discussion of IoT Edge installation and included to clarify reiterate the scope of what IS+KS+CS provide.  Although it is repeating parts from the page intro I thought it would be useful to pre-empt questions someone might mentally ask what they'd get if they only used IS+KS+CS.